### ISSUE TITLE
Button attribute doesn't take into account base classes members that are private

### ISSUE THREAD

**Issue by pawelsalwa** [dbrizov/NaughtyAttributes#228]
Button attribute omits private methods in base classes.

**Comment by dbrizov** [comment]
Can you be more specific? I tested this case and it's working fine. I attached the `NaughtyComponent` to a `GameObject` and when I press the button it logs `3.14`
```csharp
public class BaseMono : MonoBehaviour
{
	private float myFloat = 3.14f;

	[Button]
	private void Method()
	{
		Debug.Log(myFloat);
	}
}

public class NaughtyComponent : BaseMono
{
}
```

**Comment by dbrizov** [comment]
You should update to version 2.0.9. This was actually fixed today with this pull request #227

**Comment by pawelsalwa** [comment]
It's fixed I confirm. Thank you very much! :)
